qp_qiime2/tests/test_qiime2.py

```diff
@@ -122,8 +126,8 @@ def test_beta(self):
                 'plain_text')]
         self.assertEqual(ainfo[0].files, exp)
 
-        params['p-metric'] = 'unweighted UniFrac'
-        params['i-tree'] = join(
+        params['Diversity metric'] = 'Unweighted UniFrac'
+        params['Phylogenetic tree'] = join(
```

Comment:
This feels inconsistent. The tree is a file path, but the BIOM table is an internal ID?

Reply:
The inconsistency on this is confusing but I guess it is what it is.